### PR TITLE
[NL] feedback link updates

### PR DESCRIPTION
**Pull request by chejennifer**
- pass debug info to the form
- for the prefilled query chain answer, also include the url to play the query chain

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Nice!

File: static/js/utils/nl_interface_utils.ts [hunk 1]
```diff
@@ -36,16 +37,56 @@ export function isNlInterface(): boolean {
...
+      rankingClassification: debugData["ranking_classification"],
+      overviewClassification: debugData["overview_classification"],
+      sizeTypeClassification: debugData["size_type_classification"],
+      timeDeltaClassification: debugData["time_delta_classification"],
+      comparisonClassification: debugData["comparison_classification"],
+      containedInClassification: debugData["contained_in_classification"],
+      correlationClassification: debugData["correlation_classification"],
+      eventClassification: debugData["event_classification"],
+      svScores,
+    };
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/js/utils/nl_interface_utils.ts:72]: done

File: static/js/utils/nl_interface_utils.ts [hunk 2]
```diff
@@ -36,16 +37,56 @@ export function isNlInterface(): boolean {
...
+    if (Array.isArray(debugData["data_spec"])) {
+      queryChain = debugData["data_spec"].map(
+        (utterance) => utterance["query"] || ""
+      );
+    }
+    queryChain = queryChain.reverse();
+  }
+  const sourceUrl = window.location.toString().split("#")[0];
+  const queryChainUrl = queryChain.length
+    ? `(${sourceUrl}#q=${queryChain.join(";")})`
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/js/utils/nl_interface_utils.ts]: done

**Comment by chejennifer** [comment]
Merging to get this in before NL query time & can address additional comments in a followup PR

**Pull request merged**